### PR TITLE
Issue 3039

### LINKED ISSUE #3039: Updating the Additional Supporters sections of our supporters / colaboradores / nos-soutiens / apoiadores pages
Adding former (non-renewing) IPPs to the [Additional Supporters section](https://programminghistorian.org/en/supporters#additional-supporters) of our /en/supporters, /es/colaboradores, /fr/nos-soutiens, /pt/apoiadores pages.

Additional Supporters are defined as _organisations that have or continue to give support_. This enables us to acknowledge our thanks for the former support of:

- [Institute of Historical Research Wohl Library](https://www.history.ac.uk/library-digital), United Kingdom [2020].
- [Software Sustainability Institute](https://www.software.ac.uk/), United Kingdom [2021].
- [Center for Spatial and Textual Analysis (CESTA)](https://cesta.stanford.edu/), Stanford University, United States [2021].
- [Transkribus - READ COOP](https://readcoop.eu/), Austria [2022].
- [RStudio/Posit](https://posit.co/), United States [2022].

This update will also ensure that the headline list of IPPs is current and up-to-date.

### PR DESCRIPTION
**Pull request by anisa-hawes**
I'm moving former (non-renewing) IPPs to the [Additional Supporters section](https://programminghistorian.org/en/supporters#additional-supporters) of our /en/supporters, /es/colaboradores, /fr/nos-soutiens, /pt/apoiadores pages.

Closes #3039 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Assistant @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [ ] Assign at least one individual or team to "Reviewers"
  - ~~[ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.~~

### CODE REVIEW

**Comment by anisa-hawes** [comment]
Hello @charlottejmc,

When you have time, may I ask for your help to check my French on the following page?:
[ph-preview.netlify.app/fr/nos-soutiens](https://deploy-preview-3040--ph-preview.netlify.app/fr/nos-soutiens).

What I've done is: 

- Added a new heading Alumni Partners (_Partenaires des anciens élèves_) 
- Added a description to mean roughly Former contributors to our Institutional Partner Programme (_Les institutions suivantes ont participé de notre programme de Partenariat institutionnel_). Does this sound right?
- Added two sub-headings under Additional Supporters (_Soutien additionnel_) which I'd like to read the equivalent of Active and Past. Would _Actifs_ and _Passé_ seem right?

Thank you 🙂

**Comment by charlottejmc** [comment]
Hi @anisa-hawes,

I have had a look and here are my thoughts:

- _Partenaires des anciens élèves_ sounds more like _Partners of alumni_. I am thinking _Anciens élèves partenaires_ instead – but this is a tricky phrase to translate.
- This is nearly right: _Les institutions suivantes ont participé **à** notre programme de Partenariat institutionnel_ is the only correction I can see!
- I think this could work as it is, although _Passé_ would just need to be in the plural form: _Passés_. However, I might suggest something along the lines of _En cours_ and _Révolus_, which could sound more natural? Again, I feel like these are tricky phrases to translate directly.

I hope this helps!

**Comment by spapastamkou** [comment]
hi, if I may, for the term "alumni partners", simply "anciens partenaires" ou "partenariats révolus" (as Charlotte proposes) can work here. (as élèves refers to students, and I believe is embarked from the auto translation of the initial alumni term)

**Comment by anisa-hawes** [comment]
Hello @DanielAlvesLABDH,

I hope you're well. This PR represents some adjustments to the headings and sub-headings on our Supporters pages. Our aim is to create meaningful sub-headings within our lists of IPP, so we make clear who our active supporters are while also acknowledging institutions that have supported us in the past. May I ask for your help to adjust my attempted Portuguese translations?

- I added a new heading Alumni Partners (Parceiros ex-alunos? or Antigos parceiros?)
- I added a description to mean roughly Former contributors to our Institutional Partner Programme (Antigos contribuintes para o nosso Programa de Parceria Institucional)
- I added two sub-headings under Additional Supporters (Apoiadores adicionais) which I'd like to read the equivalent of Active and Past. Would Ativo and Anterior be appropriate?

You can [review the detail of my edits here](https://github.com/programminghistorian/jekyll/pull/3040/files). 

With thanks! Anisa

**Comment by DanielAlvesLABDH** [comment]
Hi @anisa-hawes, will look at this tomorrow morning. Best

File: pt/apoiadores.md [hunk 1]
```diff
@@ -59,24 +54,36 @@ Contribuintes para o nosso [Programa de Parceria Institucional](/pt/ppi).
 
 Aceitamos consultas de possíveis contribuintes para o [Programa de Parceria Institucional](/pt/ppi).
 
-## Apoiadores adicionais
+## Parceiros ex-alunos
```
Comment (DanielAlvesLABDH) [inline pt/apoiadores.md]: Anteriores parceiros

File: pt/apoiadores.md [hunk 2]
```diff
@@ -59,24 +54,36 @@ Contribuintes para o nosso [Programa de Parceria Institucional](/pt/ppi).
...
+- [Roy Rosenzweig Center for History and New Media, George Mason University](https://rrchnm.org/), Estados Unidos [2011-2022].
+- [UCL Centre for Digital Humanities](https://www.ucl.ac.uk/digital-humanities/), Reino Unido [2021-2022].
+- [Institute of Historical Research Wohl Library](https://www.history.ac.uk/library-digital), Reino Unido [2020].
+- [Software Sustainability Institute](https://www.software.ac.uk/), Reino Unido [2021].
+- [Center for Spatial and Textual Analysis (CESTA)](https://cesta.stanford.edu/), Stanford University, Estados Unidos [2021].
+- [Transkribus - READ COOP](https://readcoop.eu/), Áustria [2022].
+- [RStudio/Posit](https://posit.co/), Estados Unidos [2022].
 
+
+## Apoiadores adicionais
```
Comment (DanielAlvesLABDH) [inline pt/apoiadores.md]: Parceiros adicionais

**Review by DanielAlvesLABDH — CHANGES_REQUESTED** [review]
Have made some suggestions of change

**Comment by anisa-hawes** [comment]
Hello @spapastamkou and @rivaquiroga 

This PR is ready, with each of the translations checked by native speakers. 

The only further change I've made is to update the information about which month we reset partnership rates. As discussed with @drjwbaker and @spapastamkou, going forwards will be reviewing our rates mid-year for reset in October (rather than January) to align with ProgHist Ltd's financial year. 

Thank you.

**Review by rivaquiroga — APPROVED** [review]

**Review by DanielAlvesLABDH — APPROVED** [review]

**Pull request merged**